plone/app/locales/locales/sl/LC_MESSAGES/plone.po

```diff
@@ -8559,10 +8549,10 @@ msgstr "Lahko se zgodi, da obstajajo še relacije, ki so posledica orodja  Lingu
 msgid "description_migration_results"
 msgstr "Tu boste videli rezultate procesa migracije"
 
-#. Default: "This form allows you to directly edit the XML representation of the field list. This makes it possible to add annotations for fieldsets, validation and widgets. See the <a href=\"https://docs.plone.org/external/plone.app.dexterity/docs/reference/dexterity-xml.html\">Dexterity XML</a> section of the <a href=\"https://docs.plone.org/external/plone.app.dexterity/docs/index.html\">Dexterity Developer Manual</a> for details."
+#. Default: "This form allows you to directly edit the XML representation of the field list. This makes it possible to add annotations for fieldsets, validation and widgets. See the <a href=\"http://docs.plone.org/external/plone.app.dexterity/docs/reference/dexterity-xml.html\">Dexterity XML</a> section of the <a href=\"http://docs.plone.org/external/plone.app.dexterity/docs/index.html\">Dexterity Developer Manual</a> for details."
 #: plone.app.dexterity/plone/app/dexterity/browser/modeleditor.pt:25
 msgid "description_model_edit"
-msgstr "Ta obrazec omogoča, da neposredno urejate XML opis seznama polj. S tem je omogočeno dodajanje opomb za polja, za določbe obsega vrednosti in za gradnike. Glejte podrobnosti v <a href=\"https://docs.plone.org/external/plone.app.dexterity/docs/reference/dexterity-xml.html\">razdelek Dexterity XML</a> v priročniku <a href=\"http://docs.plone.org/external/plone.app.dexterity/docs/index.html\">Dexterity Developer Manual</a>."
+msgstr "Ta obrazec omogoča, da neposredno urejate XML opis seznama polj. S tem je omogočeno dodajanje opomb za polja, za določbe obsega vrednosti in za gradnike. Glejte podrobnosti v <a href=\"http://docs.plone.org/external/plone.app.dexterity/docs/reference/dexterity-xml.html\">razdelek Dexterity XML</a> v priročniku <a href=\"http://docs.plone.org/external/plone.app.dexterity/docs/index.html\">Dexterity Developer Manual</a>."
```

Comment:
Can you please update this one. The links are now in https

Reply:
Updated it.